src/abstractions/platformUtils.service.ts

```diff
@@ -19,6 +19,7 @@ export abstract class PlatformUtilsService {
      */
     lockTimeout: () => number;
     launchUri: (uri: string, options?: any) => void;
+    closeCurrentTab: () => void;
```

Comment:
Does this need to be a global util? Seems like you will just have a bunch of no-ops. Could you just add this to the browser extension's popup-utils?